### ISSUE TITLE
"Add Image" button in the campaign creation form is not working.

### ISSUE THREAD

**Issue by hafizrahman** [woocommerce/google-listings-and-ads#2652]
Starting from around October 14th p91TBi-clg-p2, we discovered that the "Add Image" button in the campaign creation form is no longer working. We expect the button to open an image picker, but instead it doesn't do anything.

At first we thought this is specific to mobile apps's webview, but upon testing further I see that this also happens on the Web, video below with console log (start around 0:22 for the button clicking test). This happens whether there's already existing image on a field or not:

https://github.com/user-attachments/assets/fa6dbeef-6da3-477e-8e2d-78385b9162ea

**Comment by mikkamp** [comment]
Thanks for sharing the details. I did some testing and eventually was able to reproduce the problem. Apparently it only happens when WooPayments plugin is active (even without setting up).

I ran some tests in the console and it seems that underscore is being replaced by lodash, which works fine for some functions but based on the error message it seems the `contains` function is not available when this is replaced.

We don't call `_.contains` directly in our code, but we load the WordPress media models which tries to call `_.contains` and fails with the error:
![Image](https://github.com/user-attachments/assets/8aa10569-22f4-4cbe-a590-e6babdf22e01)

From scanning the WooPayments code it looks like this might be the line where underscore is being replaced by lodash:
https://github.com/Automattic/woocommerce-payments/blob/e42103b75cca59f7cd360026915e64cbd6128afb/multi-currency/client/setup/tasks/add-currencies-task/index.js#L8
